src/components/MarkdownLayout/ModuleHeaders/ModuleHeaders.tsx

```diff
@@ -37,13 +37,17 @@ export default function ModuleHeaders({
 
   const lang = useUserLangSetting();
   const setLang = useSetUserLangSetting();
-
+  let problemIDs = [] as string[];
```

Comment:
```suggestion
  let problemIDs: string[] = [];
```

though I guess either is fine